webvtt.go

```diff
@@ -357,29 +358,80 @@ func parseTextWebVTT(i string) (o Line) {
 			}
 
 		case html.TextToken:
-			if s := strings.TrimSpace(string(tr.Raw())); s != "" {
-				// Get style attribute
-				var sa *StyleAttributes
-				if len(webVTTTagStack) > 0 {
-					tags := make([]WebVTTTag, len(webVTTTagStack))
-					copy(tags, webVTTTagStack)
-					sa = &StyleAttributes{
-						WebVTTTags: tags,
-					}
-					sa.propagateWebVTTAttributes()
+			s := string(tr.Raw())
+			// Get style attribute
+			var sa *StyleAttributes
+			if len(webVTTTagStack) > 0 {
+				tags := make([]WebVTTTag, len(webVTTTagStack))
+				copy(tags, webVTTTagStack)
+				sa = &StyleAttributes{
+					WebVTTTags: tags,
 				}
-
-				// Append item
-				o.Items = append(o.Items, LineItem{
-					InlineStyle: sa,
-					Text:        unescapeWebVTT(s),
-				})
+				sa.propagateWebVTTAttributes()
 			}
+
+			// Append item
+			items := parseTextWebVTTText(sa, s)
+			o.Items = append(o.Items, items...)
 		}
 	}
 	return
 }
 
+func parseTextWebVTTText(sa *StyleAttributes, line string) []LineItem {
+	var ret []LineItem
+
+	// split the line by inline timestamps
+	indexes := inlineTimestampRegexp.FindAllStringSubmatchIndex(line, -1)
```

Comment:
Could you first check `if len(indexes) == 0` instead and return in the if block? That will allow to remove the `else`